### ISSUE TITLE
Can not use arrow key to navigate when create project inside an existing directory

### ISSUE THREAD

**Issue by zmen** [vuejs/vue-cli#4181]
### Version
3.8.4

### Environment info
```
  System:
    OS: macOS 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
  Binaries:
    Node: 12.4.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 75.0.3770.100
    Firefox: 67.0
    Safari: 12.1.1
  npmGlobalPackages:
    @vue/cli: 3.8.4
```

### Steps to reproduce
1. create an empty directory by `mkdir demo` 
2.  `cd demo`
3. run `vue create .`
4. Prompt `Generate project in current directory?`, answer `Y`
5. It will ask you to pick a preset

### What is expected?
I can pick a preset by arrow keys.

### What is actually happening?
Nothing happened after press arrow key.
Even `Cmd   c` can't escape the terminal.

![Jun-21-2019 12-34-10](https://user-images.githubusercontent.com/11732386/59897911-57d63b00-9421-11e9-9685-3307a7b03a7c.gif)

**Comment by MRVDH** [comment]
I'm experiencing example the same problem on:
Windows 10
vue cli 3.8.4
node.js v10.12.0

**Comment by MRVDH** [comment]
Upstream bug due to SBoudrias/Inquirer.js#811

**Comment by sodatea** [comment]
Fixed in inquirer@6.4.1
Now please try to clean your npm/yarn cache and reinstall `@vue/cli`
